packages/jaeger-ui/src/components/App/TopNav.test.js

```diff
@@ -30,6 +30,7 @@ describe('<TopNav>', () => {
     {
       label: 'Twitter',
       url: 'https://twitter.com/JaegerTracing',
+      openInSameTab: true,
```

Comment:
does this make a difference?

Reply:
In `renders the nested menu items` test, there is assertion if this is passed further.
I tried to make a test for rendered dropdown, but I just don't know, how to do it.